### ISSUE TITLE
partial tactile paving

### ISSUE THREAD

**Issue by andrewharvey** [westnordost/StreetComplete#2260]
The https://wiki.openstreetmap.org/wiki/Key:tactile_paving quest can be unclear for users where only one side of the crossing has tactile paving. This is quite common, from a random survey today I'd say about 20% might have been like this.

From the wiki though it's unclear how to actually tag this, perhaps `partial` as the value.

**Comment by westnordost** [comment]
Unclear? The quest asks: 

_Does this crosswalk have tactile pavings **on both sides**?_ (emphasis mine)

**Comment by rouelibre1** [comment]
I found the app quite clear and the quest itself not confusing. It's a simple binary answer. I only got confused after looking at the wiki page, just like @andrewharvey (which most users will probably not visit)

From what I understand from the wiki page, the best thing to do for crosswalks with tactile paving on only one side is to tag them with `tactile_paving=no` in cases where we want to provide an anwser, which is what the app does.
